cypress/integration/stoplight-project.ts

```diff
@@ -7,18 +7,18 @@ describe('Stoplight component', () => {
     loadStoplightProjectPage();
     cy.findByText('pets').click();
     cy.findByText('Create a pet').click();
-    cy.findByRole('heading', { name: /Create a pet/i }).should('exist');
+    cy.findByRole('heading', { name: /Create a pet/i, timeout: 30000 }).should('exist');
```

Comment:
Increased the timeout until this PR is merged: https://github.com/stoplightio/platform-internal/pull/6409

Reply:
It got merged! :D